[CAI-119] Add chat message convert md to html (#1044)
pagopa/developer-portal: .changeset/spotty-wasps-approve.md

---
"nextjs-website": patch
"storybook-app": patch
---

> Align chat message to mock add parser to convert md into html

Reviewer: ```suggestion
Add a parser to convert chatbot messages from Markdown to HTML format
```
